### PR TITLE
feat: add temporary login

### PR DESCRIPTION
**Pull request by ethanwu10**
Reopening #29 (prematurely merged by @kmh11)

Copied from #29:

this is apparently extremely important, toggleable by `enableLogin` in yml

same concerns with field label as in https://github.com/climatEDU/climatEDU/pull/22#issuecomment-683229848

### CODE REVIEW

**Comment by kmh11** [comment]
why is the button skinny again

**Comment by kmh11** [comment]
update: the button is fat again

**Review by kmh11 — APPROVED** [review]
lgtm

File: src/pages/login.js [hunk 1]
```diff
@@ -1,61 +1,80 @@
...
-        </Button>
-        <Text sx={{ color: 'highlight', marginTop: 20, textAlign: 'center' }}>
-          {error}
-        </Text>
-      </Box>
-    </Box>
+          <Button onClick={onSubmit} sx={{ mb: 3 }}>
+            Submit
+          </Button>
+          <Text sx={{ color: 'red' }}>{error}</Text>
```
Comment (ethanwu10) [inline src/pages/login.js:72]:
```suggestion
          <Text sx={{ color: 'error' }}>{error}</Text>
```

**Review by ethanwu10 — COMMENTED** [review]
Toast containers are in master, please implement toasts in this PR instead of using (inaccessible) message box.

File: src/pages/login.js [hunk 2]
```diff
@@ -1,61 +1,74 @@
...
+            onChange={e => setEmail(e.target.value)}
+            sx={{ mb: 2 }}
           />
-        </Button>
-        <Text sx={{ color: 'highlight', marginTop: 20, textAlign: 'center' }}>
-          {error}
-        </Text>
-      </Box>
-    </Box>
+          <Button onClick={onSubmit}>
```
Comment (ethanwu10) [inline src/pages/login.js:65]: The event handler should be attached to the form submit event, not the click event on the button. This change will improve a11y and keyboard usability. The same issue was previously brought up [here](https://github.com/climatEDU/climatEDU/pull/22#discussion_r485228624).

File: src/pages/login.js [hunk 3]
```diff
@@ -1,61 +1,74 @@
...
-  }
 
-  const user = useAuth()
-  if (user !== null) {
-    return <Redirect to='/' noThrow />
+    /* TODO: use toasts? */
+    try {
+      await firebaseApp.firestore().collection('reminders').add({ email })
+      toast.success('Submitted!')
+      setEmail('')
```
Comment (ethanwu10) [inline src/pages/login.js:30]: Consider redirecting.

File: src/pages/login.js [hunk 4]
```diff
@@ -1,61 +1,76 @@
+/** @jsx jsx */
 import { useState } from 'react'
-import { Redirect } from '@reach/router'
-import { jsx, Box, Text, Heading } from 'theme-ui'
-import Button from '../components/button'
+import { Button, Flex, Input, Heading, Text, jsx } from 'theme-ui'
+import { navigate } from "gatsby"
```
Comment (ethanwu10) [inline src/pages/login.js:4]: Consider `useNavigate` over global `navigate`

File: src/pages/login.js [hunk 5]
```diff
@@ -1,61 +1,76 @@
...
+    if (!/\S+@\S+\.\S+/.test(email)) {
+      toast.error('Invalid email.')
       return
     }
-  }
 
-  const user = useAuth()
-  if (user !== null) {
-    return <Redirect to='/' noThrow />
+    /* TODO: use toasts? */
```
Comment (ethanwu10) [inline src/pages/login.js:27]:
```suggestion
```

File: src/pages/login.js [hunk 6]
```diff
@@ -1,61 +1,76 @@
...
-            sx={{
-              color: 'primary',
-              verticalAlign: 'middle',
-              marginLeft: 10,
-            }}
+        <Flex
+          as='form'
+          onSubmit={onSubmit}
+          sx={{
+            maxWidth: 300,
```
Comment (ethanwu10) [inline src/pages/login.js:56]: Prefer a style prop.

**Review by ethanwu10 — COMMENTED** [review]
lgtm, should probably fix width value before merging

**Pull request merged**